lib/ingram_micro/elements/message_header_no_pw.rb

```diff
@@ -16,8 +16,12 @@ def defaults
   end
 
   def valid?
-    raise IngramMicro::InvalidType.new('message_id must be a number') unless integer?(@element[:message_id])
-    raise IngramMicro::MissingField.new('partner_name must be present') unless @element[:partner_name]
+    unless integer?(@element[:message_id])
```

Comment:
yeah i agree, plus i think code climate will complain about the multiline statement. also, i don't like `unless`